jcl/src/java.base/share/classes/module-info.java.extra

```diff
@@ -21,21 +21,52 @@
  * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
  *******************************************************************************/
 
-exports com.ibm.sharedclasses.spi to openj9.sharedclasses, java.management, java.rmi;
-exports com.ibm.oti.vm to java.management, jdk.attach, jdk.management, openj9.cuda, openj9.gpu, openj9.jvm, openj9.sharedclasses;
-exports com.ibm.oti.util to java.management, jdk.attach, jdk.jcmd, jdk.management, openj9.sharedclasses;
-exports openj9.internal.tools.attach.target to jdk.attach, jdk.jcmd, jdk.management;
-exports openj9.management.internal to java.management, openj9.jvm;
-exports openj9.internal.management to java.management;
-exports jdk.internal.org.objectweb.asm to openj9.dtfj, openj9.dtfjview;
+exports com.ibm.sharedclasses.spi to
+	java.management,
+	java.rmi,
+	openj9.sharedclasses;
```

Comment:
Leading tabs may pose a problem. In the past, the tooling in the extension repos only supported spaces for indentation.

Reply:
Is there a link to the issue, or platform specific? It works at MacOS & Linux AMD, there was no nightly builds yesterday due to 0.32 release.

Reply:
Actually, the dependency detection code still demands that `requires` clauses not be preceded by any tabs.
Of course, this doesn't apply to `java.base` (which can't require any other module), but we should probably still avoid tabs in `module-info.java` files.

Reply:
You can see the dependency problem for yourself:
1. run `make openj9.gpu-java`
2. notice that `make-support/module-deps.gmk` says `DEPS_openj9.gpu := java.base openj9.cuda`
3. replace the leading spaces in `jcl/src/openj9.gpu/share/classes/module-info.java` with tabs
4. run `make openj9.gpu-java` again
5. notice that `make-support/module-deps.gmk` says `DEPS_openj9.gpu := java.base`

Summary: don't use tabs in `module-info.*` files.

Reply:
The tab wasn't recognized because of the script - https://github.com/ibmruntimes/openj9-openjdk-jdk/blob/e27dba6fdc57241993cd4daedaac3d6bab71f026/make/common/Modules.gmk#L181-L193
This `module-info.java.extra` intended to match OpenJDK counterpart which uses spaces for the qualified exports.
Created https://github.com/eclipse-openj9/openj9/pull/14947.